src/Features/Persistence/Blockcore.Features.Base.Persistence.LevelDb/LevelDbChainStore.cs

```diff
@@ -1,15 +1,17 @@
 ﻿using System;
 using System.Collections.Generic;
 using Blockcore.Configuration;
+using Blockcore.Consensus;
 using Blockcore.Consensus.BlockInfo;
+using Blockcore.Consensus.Chain;
 using Blockcore.Networks;
 using Blockcore.Utilities;
 using LevelDB;
 using NBitcoin;
 
-namespace Blockcore.Consensus.Chain
```

Comment:
Is this the right place to place the folder Persistence (under features)

Reply:
it's technically a feature part, the feature is the parent of these project, without a feature that part can't exists (so that's why it's a feature subfolder), what's your suggestion?

Reply:
ok fair point I agree it is a sub feature so it makes sense here